ember-cli-build.js

```diff
@@ -6,7 +6,12 @@ module.exports = function(defaults) {
     var app = new EmberApp(defaults, {
         'ember-bootstrap': {
             importBootstrapFont: false
-        }
+        },
+
+        babel: {
+            optional: ['es6.spec.symbols'],
```

Comment:
Standard question: what is the impact of this polyfill on final production build size? 

(stats reported from `rm -rf dist/ && ember build --environment production`)

This won't matter much now- but once we start converting the monolithic OSF, we want to be in the habit of policing build size very aggressively, as the dependencies and polyfills will really start to add up.